python/paddle/fluid/tests/unittests/test_imperative_resnet.py

```diff
@@ -277,7 +277,7 @@ def test_resnet_float32(self):
 
                 dy_grad_value = {}
                 for param in resnet.parameters():
-                    if not param.stop_gradient:
+                    if param.trainable:
```

Comment:
what's the relation between trainable and stop_gradient.

Reply:
stop_gradient controls the gradient flow and trainable controls whether the parameter could be trained properly.